src/js/components/Tip/Tip.js

```diff
@@ -21,6 +21,90 @@ const Tip = forwardRef(
 
     const componentRef = useForwardedRef(tipRef);
 
+    // if margin is a string, parse it or use the theme value
+    const normalizeMargin = (margin) => {
+      if (typeof margin === 'string') {
+        const value = theme.global.edgeSize[margin] || parseInt(margin, 10);
+        return { top: value, bottom: value, left: value, right: value };
+      }
+      return { top: 0, bottom: 0, left: 0, right: 0 };
+    };
+
+    const tipMarginStyle = (theme, align) => {
```

Comment:
passing this theme in is causing a `no-shadow` lint error but when I take it out its not working property any devs have thoughts?